### PR TITLE
feat(controller): send events to Nginx resource about Service reconciling

### PR DESCRIPTION
**Pull request by nettoclaudio**
:zap:

### CODE REVIEW

File: controllers/nginx_controller.go [hunk 1]
```diff
@@ -149,23 +152,27 @@ func (r *NginxReconciler) reconcileDeployment(ctx context.Context, nginx *nginxv
...
 	var currentService corev1.Service
-	err := r.Client.Get(ctx, svcName, &currentService)
-	if err != nil && errors.IsNotFound(err) {
-		logger.
-			WithValues("ServiceResource", newService).V(4).Info("Creating a Service resource")
+	err := r.Client.Get(ctx, types.NamespacedName{Name: newService.Name, Namespace: newService.Namespace}, &currentService)
+
+	if errors.IsNotFound(err) {
+		err = r.Client.Create(ctx, newService)
+		switch err {
```
Comment (wpjunior) [inline controllers/nginx_controller.go:162]: why not using classic if-else ?

**Comment by wpjunior** [comment]
awesome!

File: controllers/nginx_controller.go (hunk 1 above)
Comment (nettoclaudio) [reply controllers/nginx_controller.go:162]: Just for personal preference :grimacing:

Comment (nettoclaudio) [reply controllers/nginx_controller.go:162]: Is it wordy?

Comment (tcarreira) [reply controllers/nginx_controller.go:162]: I though the same as @wpjunior 😅 
As there is only 1 switch and 1 default, I would go for a if-else

Comment (wpjunior) [reply controllers/nginx_controller.go:162]: if-else is simpler than switch.

Comment (nettoclaudio) [reply controllers/nginx_controller.go:162]: I did it to ease future mappings events from errors. However for now as we've a single mapping... it really doesn't do much sense.

Comment (nettoclaudio) [reply controllers/nginx_controller.go]: Did it

**Pull request merged**